bin/rbenv-update

```diff
@@ -26,33 +27,57 @@ else
   reset=""
 fi
 
-indent_output() {
-  while read data; do
-    printf " ${color}|${reset}  %s\n" "$data"
-  done
+# Helper functions
+pretty_output() {
+  if [[ -n "$verbose" ]]; then
+    while read -r data; do
+      printf " ${color}|${reset}  %s\n" "$data"
+    done
+  else
+    cat >/dev/null 2>&1
+  fi
+}
+
+print_colored() {
+  if [[ -z "$silent" ]]; then
+    printf "${color}%s${reset}\n" "$1"
+  fi
+}
+
+git() {
+  if [[ -n "$noop" ]]; then
+    echo git "$@"
+  else
+    command git "$@" | pretty_output
+  fi
 }
 
 is_rbenv_git_repo() {
-  git remote -v 2>/dev/null | grep -q 'rbenv\|ruby-build'
+  command git remote -v 2>/dev/null | grep -q 'rbenv\|ruby-build'
```

Comment:
good call. probably need `command` added to the git-pull invocation as well a few lines down?